streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorTopologyTest.java

```diff
@@ -1569,24 +1064,6 @@ private Topology createSimpleTopologyWithDroppingPartitioner() {
                 .addSink("sink", OUTPUT_TOPIC_1, new DroppingPartitioner(), "processor");
     }
 
-    @Deprecated // testing old PAPI
-    private Topology createStatefulTopology(final String storeName) {
-        return topology
-            .addSource("source", STRING_DESERIALIZER, STRING_DESERIALIZER, INPUT_TOPIC_1)
-            .addProcessor("processor", define(new OldAPIStatefulProcessor(storeName)), "source")
-            .addStateStore(Stores.keyValueStoreBuilder(Stores.inMemoryKeyValueStore(storeName), Serdes.String(), Serdes.String()), "processor")
-            .addSink("counts", OUTPUT_TOPIC_1, "processor");
-    }
-
-    @Deprecated // testing old PAPI
-    private Topology createConnectedStateStoreTopology(final String storeName) {
-        final StoreBuilder<KeyValueStore<String, String>> storeBuilder = Stores.keyValueStoreBuilder(Stores.inMemoryKeyValueStore(storeName), Serdes.String(), Serdes.String());
-        return topology
-            .addSource("source", STRING_DESERIALIZER, STRING_DESERIALIZER, INPUT_TOPIC_1)
-            .addProcessor("processor", defineWithStoresOldAPI(() -> new OldAPIStatefulProcessor(storeName), Collections.singleton(storeBuilder)), "source")
```

Comment:
`defineWithStoresOldAPI()` should be unused now and can be removed, too?

Reply:
Done.